lib/ldp-middleware.js

```diff
@@ -26,7 +26,7 @@ function LdpMiddleware (corsSettings) {
   router.post('/*', allow('Append'), post)
   router.patch('/*', allow('Append'), patch)
   router.put('/*', allow('Write'), put)
-  router.delete('/*', allow('Write'), del)
+  router.delete('/*', allow('Write', '..'), del)
```

Comment:
Not sure, but wouldn't `./` be the path to the container?

Reply:
The reason why that was needed is because I used `path.join`, which considers everything to be a path segment, including the filename itself, so `./` would do nothing.

In any case, I rewrote this PR a bit, as it was indeed a bit confusing.

Reply:
OK; perhaps you can have a look too, @RubenVerborgh ?